.circleci/config.yml

```diff
@@ -471,40 +487,30 @@ jobs:
       CC: clang
       CXX: clang++
       MAKEFLAGS: -j 10
-    steps:
-      - checkout
-      - run: *run_build
-      - store_artifacts: *artifacts_solc
-      - persist_to_workspace: *artifacts_executables
+    <<: *steps_b_ubu
 
-  b_ubu_asan_clang: &b_ubu_asan_clang
-    docker:
-      - image: << pipeline.parameters.ubuntu-2004-clang-docker-image >>
-    environment:
-      CC: clang
-      CXX: clang++
-      CMAKE_OPTIONS: -DSANITIZE=address
-      MAKEFLAGS: -j 3
-    steps:
-      - checkout
-      - run: *run_build
-      - store_artifacts: *artifacts_solc
-      - persist_to_workspace: *artifacts_executables
-
-  b_ubu_ubsan_clang: &b_ubu_ubsan_clang
+  b_ubu_au_clang:
+    parameters:
+      cmake_option:
+        type: string
+      gitter_notify:
+        type: boolean
+        default: false
     docker:
       - image: << pipeline.parameters.ubuntu-2004-clang-docker-image >>
     environment:
       CC: clang
       CXX: clang++
```

Comment:
Oh, I see that it uses a different base image. So it would have to be another parameter.

Or you could leave it as is but instead unify `b_ubu_au_clang` and `b_ubu_clang`. Adding an empty `CMAKE_OPTIONS` in `b_ubu_clang` should not hurt.